docs/_posts/examples/3400-01-15-attribution-position.html

```diff
@@ -14,8 +14,7 @@
     style: 'mapbox://styles/mapbox/light-v9',
     center: [-77.04, 38.907],
     zoom: 11.15,
-    attributionControl: {
-        position: 'top-left'
-    }
+    attributionControl: false
 });
+map.addControl(new mapboxgl.AttributionControl(), 'top-left');
```

Comment:
does `AttributionControl` need to be revamped to no longer rely on `mapboxgl` or is that just external plugins?